### ISSUE TITLE
TestResource_YamlDecode and TestResource_JsonDecode fail at go test

### ISSUE THREAD

**Issue by rogers0** [viant/toolbox#30]
Hi there,

I'm working with @creekorful to package this project into Debian.
Debian build system usually run go test after building the package, so I find TestResource_YamlDecode and TestResource_JsonDecode fail, whatever has network access.
Here's the log:

```
=== RUN   TestResource_YamlDecode
    TestResource_YamlDecode: resource_test.go:97:
                Error Trace:    resource_test.go:97
                Error:          Not equal:
                                expected: "file:///build/golang-github-viant-toolbox-0.31.4/obj-x86_64-linux-gnu/src/github.com/viant/toolbox/url/resource1.yaml"
                                actual  : "file://resource1.yaml"

                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -1 +1 @@
                                -file:///build/golang-github-viant-toolbox-0.31.4/obj-x86_64-linux-gnu/src/github.com/viant/toolbox/url/resource1.yaml
                                +file://resource1.yaml
                Test:           TestResource_YamlDecode
--- FAIL: TestResource_YamlDecode (0.00s)
=== RUN   TestResource_JsonDecode
resource.json
    TestResource_JsonDecode: resource_test.go:153:
                Error Trace:    resource_test.go:153
                Error:          Not equal:
                                expected: "file:///build/golang-github-viant-toolbox-0.31.4/obj-x86_64-linux-gnu/src/github.com/viant/toolbox/url/resource.json" 
                                actual  : "file://resource.json"

                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -1 +1 @@
                                -file:///build/golang-github-viant-toolbox-0.31.4/obj-x86_64-linux-gnu/src/github.com/viant/toolbox/url/resource.json
                                +file://resource.json
                Test:           TestResource_JsonDecode
--- FAIL: TestResource_JsonDecode (0.00s)
```

**Comment by adranwit** [comment]
updated test plan

**Comment by rogers0** [comment]
I confirmed this is fixed on version 0.32.2
Thanks for your support!
